static/schemas/2.0/workflowGraphs.2.0.json

```diff
@@ -0,0 +1,59 @@
+{
+    "title": "workflowGraphs.2.0",
+    "definitions": {
+        "tasks": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "oneOf": [
+                    {"required": ["taskName"]},
+                    {"required": ["taskDefinition"]}
+                ],
+                "properties": {
+                    "label": {
+                        "type": "string",
+                        "minLength": 1
+                    },
+                    "taskName": {
+                        "type": "string",
+                        "minLength": 1
+                    },
+                    "taskDefinition": {
+                        "type": "object"
+                    },
+                    "ignoreFailure": {
+                        "type": "boolean"
+                    },
+                    "waitOn": {
+                        "type": "object"
+                    }
+                }
+            }
```

Comment:
Suggest adding "minItems: 1", so that a graph with empty task definition will be caught as invalid during schema validation